src/components/terminal/SimpleTerminal.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from "react";
+import Terminal from "react-console-emulator";
+
+function SimpleTerminal() {
+  const commands = {
```

Comment:
Create a new file called **Commands.js** and declare the commands var init then import into this file and use it.